ext/GraphNeuralNetworksSimpleWeightedGraphsExt/GraphNeuralNetworksSimpleWeightedGraphsExt.jl

```diff
@@ -0,0 +1,10 @@
+module GraphNeuralNetworksSimpleWeightedGraphsExt
+
+using GraphNeuralNetworks
+using Graphs
+import GraphNeuralNetworks: AbstractGraph
+using SimpleWeightedGraphs
+
+include("GNNGraphs/gnngraph.jl")
```

Comment:
let's just put everything here, no need to have a second file, I don't expect this extension to grow much